src/ggml-backend.c

```diff
@@ -246,6 +246,20 @@ GGML_CALL void ggml_backend_tensor_get(const struct ggml_tensor * tensor, void *
     buf->iface.get_tensor(buf, tensor, data, offset, size);
 }
 
+GGML_API GGML_CALL void ggml_backend_tensor_memset(struct ggml_tensor * tensor, uint8_t value, size_t offset, size_t size) {
+    ggml_backend_buffer_t buf = tensor->view_src ? tensor->view_src->buffer : tensor->buffer;
+
+    GGML_ASSERT(buf != NULL && "tensor buffer not set");
+    GGML_ASSERT(tensor->data != NULL && "tensor not allocated");
+    GGML_ASSERT(offset + size <= ggml_nbytes(tensor) && "tensor write out of bounds");
+
+    if (!size) {
+        return;
+    }
+
+    buf->iface.memset_tensor(buf, tensor, value, offset, size);
+}
```

Comment:
This would make the reason for failure on other backends more clear:

```suggestion
    if (!size) {
        return;
    }
    
    GGML_ASSERT(buf->iface.memset_tensor != NULL && "memset not supported by backend buffer");

    buf->iface.memset_tensor(buf, tensor, value, offset, size);
}
```